runtests.py

```diff
@@ -165,6 +165,7 @@ def main(argv):
         sys.path.insert(0, site_dir)
         sys.path.insert(0, site_dir_noarch)
         os.environ['PYTHONPATH'] = site_dir + os.pathsep + site_dir_noarch
+        write_type_hints(site_dir)
```

Comment:
this is causing a circular import dependency. Can you move it to after line 170?

Reply:
Sure thing, I'll fix during the next set of pushes.